src/Collection/CollectionTrait.php

```diff
@@ -269,10 +269,11 @@ public function groupBy($path): CollectionInterface
         foreach ($this->optimizeUnwrap() as $value) {
             $pathValue = $callback($value);
             if ($pathValue === null) {
-                throw new InvalidArgumentException(
-                    'Cannot use a nonexistent path or null value. ' .
-                    'Use a callback to provide default values if necessary.'
-                );
+                throw new InvalidArgumentException(sprintf(
+                    'Cannot group by path that does not exist or contains a null value. ' .
+                    'Use a callback to return a default value.',
+                    $path
```

Comment:
Psalm isn't happy about this using sprintf() with no placeholders. Perhaps these should be basic strings?

Reply:
I was printing the path name before I remembered it can be a closure. Will remove it.